src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -50,7 +50,15 @@ let idPool = 0;
 				<div class="tabContainer">
 					<div class="tabList" role="tablist" scrollable [horizontalScroll]="AutoScrollbarVisibility" [verticalScroll]="HiddenScrollbarVisibility" [scrollYToX]="true">
 						<div role="presentation" *ngFor="let tab of _tabs">
+						<ng-container *ngIf="tab.type!=='header'">
```

Comment:
Is the logic correct here? The one for tab.type!=='header' is showing the tab-header component, while the other is just the basic span which doesn't seem correct.

Reply:
renamed to group-header